src/app/3d/qgs3deditutils.h

```diff
@@ -0,0 +1,81 @@
+/***************************************************************************
+    qgs3deditutils.h
```

Comment:
After seeing the whole code, I wonder whether it makes sense to actually move code from Qgs3DMapToolPointCloudChangeAttribute to Qgs3DEditUtils - because if Qgs3DMapToolPointCloudChangeAttribute is just a based class for concrete implementations, it is pretty much empty - and the code in Qgs3DEditUtils could simply stay there... (I don't think it would be needed in other parts of QGIS code anytime soon)